### PR TITLE
Process form 4

### LINKED ISSUE #253: Form 4 Parser
**Is your feature request related to a problem? Please describe.**
Insider trading information has become a hot commodity for retail investors. The parser does not currently extract pertinent data on buying and selling of securities by insiders.

**Describe the solution you'd like**
Add a function to the parser to extract non-derivative transaction information pertinent to retail investors intending to track trades of insiders.

**Describe alternatives you've considered**
All other alternatives are closed-source/proprietary and paid.

**Additional context**
No additional context.

### PR DESCRIPTION
**Pull request by darrion**
- [x] closes #253 
- [x] tests added / passed
- [x] passes `git diff upstream/master -u -- "*.py" | flake8 --diff`
- [x] added entry to docs/whatsnew latest version

### CODE REVIEW

**Comment by darrion** [comment]
Resolved all linter warnings, executed tests, and confirmed branch is up to date with master.

File: secedgar/parser.py [hunk 1]
```diff
@@ -221,3 +240,100 @@ def process_document_metadata(doc):
...
         return metadata_doc
+
+
+class F4Parser:
+    """Utility class to extract actionable data and documents from a single text file.
+
+    .. warning::
+        The ``F4Parser`` class is still experimental. Use with caution.
+
+    .. versionadded:: 0.3.4
```
Comment (jackmoody11) [inline secedgar/parser.py:251]: This should be 0.4.0

**Review by jackmoody11 — CHANGES_REQUESTED** [review]
Only change needed is to change version added in doc string to 0.4.0.

File: secedgar/parser.py (hunk 1 above)
Comment (darrion) [reply secedgar/parser.py]: Corrected it.

**Review by jackmoody11 — APPROVED** [review]

**Comment by jackmoody11** [comment]
Good work @darrion. Thanks for your help!

**Pull request merged**